client-v2/src/bundles/frontsUIBundle.ts

```diff
@@ -213,12 +256,43 @@ const clearArticleFragmentSelection = (state: State, frontId: string) => ({
 
 const reducer = (state: State = defaultState, action: Action): State => {
   switch (action.type) {
+    case EDITOR_OPEN_CURRENT_FRONTS_MENU: {
+      return {
+        ...state,
+        showOpenFrontsMenu: true
+      };
+    }
+
+    case EDITOR_CLOSE_CURRENT_FRONTS_MENU: {
+      return {
+        ...state,
+        showOpenFrontsMenu: false
+      };
+    }
+
     case EDITOR_OPEN_FRONT: {
       return {
         ...state,
         frontIds: state.frontIds.concat(action.payload.frontId)
       };
     }
+    case EDITOR_MOVE_FRONT: {
+      const maxIndex = state.frontIds.length - 1;
+      if (
+        state.frontIds.indexOf(action.payload.frontId) === -1 ||
```

Comment:
minor: would this be more readable if we extract them to local variables?

Reply:
good point, I've just done this 👍